### ISSUE TITLE
Empty value trigger segfault in label with LABELCACHE OFF

### ISSUE THREAD

**Issue by yjacolin** [mapserver/mapserver#5241]
Here is a minimalist mapfile from wxs msautotest:
```
MAP

NAME WMS_TEST
SHAPEPATH ./data
FONTSET etc/fonts.txt

WEB
  METADATA
    "wms_onlineresource"   "http://localhost/path/to/ows?"
    "ows_enable_request" "DescribeLayer" 
    "wms_enable_request" "!GetCapabilities !GetFeatureInfo GetMap"
  END
END

PROJECTION
  "init=epsg:4326"
END

LAYER
  NAME lakes_elev

  METADATA
    "DESCRIPTION" "Lakes with dimesions (elevation and text)"
     "wms_title"         "Lakes"
    "wms_description"   "Lakes"
    "wms_enable_request" "*"
  END
  TYPE POLYGON
  STATUS ON
  DATA lakes2
   PROJECTION
    "init=epsg:4326"
  END
  LABELITEM "text_dimen"
  LABELCACHE OFF # <---- needed to triger the issue
  CLASS
    NAME "Lakes with elevation"
    STYLE
      COLOR 0 0 240
      OUTLINECOLOR 0 0 199    
    END
    LABEL
      COLOR 0 0 0
      OUTLINECOLOR 255 255 255
      FONT "vera"
      TYPE truetype
      SIZE 8
      POSITION cc
        STYLE
          COLOR 255 0 0
        END
    END #end label
  END
END # layer

END
```
/!\ you need the lakes2.shp with an empty value in text_dimen as the problem come from LABELCACHE OFF with an empty value in the column.

**Comment by tbonfort** [comment]
@yjacolin please send me your lakes2 dataset privately

**Comment by tbonfort** [comment]
Fixed in 9ade01a

**Comment by yjacolin** [comment]
thanks, this is fixed indeed.
